example/tests/test_filters.py

```diff
@@ -338,3 +338,33 @@ def test_filter_missing_rvalue_equal(self):
         dja_response = response.json()
         self.assertEqual(dja_response['errors'][0]['detail'],
                          "missing filter[headline] test value")
+
+    def test_search_keywords(self):
+        """
+        test for `filter[search]=keyword1...` (keyword1 [AND keyword2...])
+        """
+        for keywords in ("research", "chemistry", "nonesuch",
```

Comment:
This test is a bit difficult to comprehend. It would be much easier to write it with [pytest parametrize](https://docs.pytest.org/en/latest/parametrize.html).

I ask myself though do we really need a test for SearchFilter? It is DRF functionality and as such it is already tested in DRF itself and we just do same work again. Hence I think we could remove this test.